web/src/user_group_edit.js

```diff
@@ -330,10 +330,15 @@ export function handle_member_edit_event(group_id, user_ids) {
         $row.replaceWith($new_row);
     }
 
+    // This should only be triggered when a user is on another group
+    // edit panel and they join a group via the left panel plus button.
+    // In that case, the edit panel of the newly joined group should
+    // open. `is_user_in_group` with direct_members_only set to true acts
+    // as a proxy to check if it's an `add_members` event
```

Comment:
Currently, if a user has opened groups UI and viewing group A in the right panel and some other user adds that user to group B, the right panel changes to B and interrupts the flow of user who might be editing something in A.

I guess we can change the behavior to open the edit panel only when user adds themselves to the group. Thoughts?

Reply:
Fixed it. I'm now handling this logic in add_or_remove_group.